project/ReflectiveCodeGen.scala

```diff
@@ -13,20 +13,23 @@ import sbt.internal.inc.classpath.ClasspathUtilities
 import akka.grpc.sbt.AkkaGrpcPlugin.autoImport._
 
 import scala.collection.mutable.ListBuffer
+import protocbridge.{ Artifact => BridgeArtifact }
```

Comment:
changes in `ReflectiveCodeGen` are teh result of taking several differences from the equivalent `ReflectiveCodeGen` on akka-grpc.